spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-messaging/pom.xml

```diff
@@ -81,6 +81,10 @@
 			<artifactId>lombok</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>com.github.kristofa</groupId>
```

Comment:
this is not new. the samples that have brave added here only have to add it because their parent no-longer includes a brave dependency. they need it because of this https://github.com/openzipkin/zipkin-java/issues/68

if you print the dep tree you'll notice there's in fact less brave